linkerd/proxy/dns-resolve/src/lib.rs

```diff
@@ -84,7 +85,7 @@ async fn resolution(dns: dns::Resolver, na: NameAddr) -> Result<UpdateStream, Er
             loop {
                 match dns.resolve_addrs(na.name(), na.port()).await {
                     Ok((addrs, expiry)) => {
-                        debug!(?addrs);
+                        debug!(?addrs, name = %na);
```

Comment:
should we maybe just stick the spawned task in a debug span with the `name` field? (we could use `Span::or_current`)

that way, we would also get the nameaddr on the "Closed" and error events.

Reply:
we should usually have it on the parent span already, maybe not really necessary here